### ISSUE TITLE
uptime duration is not translated

### ISSUE THREAD

**Issue by StCyr** [PluginsOCSInventory-NG/uptime#13]
uptime duration is not translated; It is reported in French by the Windows agent and in English by the Unix agent.

This is not optimal.

A better solution would be for the agent to report in some machine format (eg: UNIX epoch) and have the duration tranlsated in ocsreports.

**Comment by sokatra** [comment]
yes ..that is right .. problem is that you can't easily search for uptime > than 15 days .. because on some computers the uptimes will be reported as 
0 jours, 0 heures, 32 minutes (this report was from server 2.8.1 and client 2.05 - uptime plugin 2.1 ' Date : 27/09/2017)
on an english windows 7- 64 bit system..but it isn't always french on any system .. on most system the plugin report in english 
8 Days, 0 Hours, 10 Minutes (the last report is from server 2.1.2 and client 2.0.5 - uptime plugin 2.0 for OCS Inventory NG 2.x from ' Version :		2.00
' Date :		24/07/2017 )

I think that the result was only send as text to the server .. not numerical

btw. under https://github.com/PluginsOCSInventory-NG/uptime/releases/tag/2.0 you will get version 2.1 not 2.0 of uptime plugin

**Comment by Lea9250** [comment]
Hi @StCyr 

I'm closing this issue since it was opened a while ago. I did some quick testing and both the computer name and printer name display correctly in the search. 

Best regards,
Léa
